Actors/Behaviours/ITurnBehaviour.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TurnBasedFeest.Actions;
+using TurnBasedFeest.Utilities;
+
+namespace TurnBasedFeest.Actors.Behaviours
+{
+    interface ITurnBehaviour
+    {
+        bool DetermineBehaviour(Input input, List<Actor> actors, Actor currentActor);
```

Comment:
This returns a boolean, could this not be better as a more descriptive thing? Like Behaviour.FINISHED or Behaviour.UI_INTERACTION?